src/app/screens/LandingScreen.js

```diff
@@ -23,7 +23,7 @@ export default class LandingScreen extends BaseScreen {
       </Section>
 
       <Section show={true}>
-        <Video sources={[{src: asset('assets/vid/landing-screen-intro.mp4'), type: 'video/mp4'}]}/>
+        <Video sources={[{src: asset('assets/vid/landing-screen-intro.mp4'), type: 'video/mp4'}]} autoplay={true}/>
```

Comment:
Don't edit my code. Remove this after you have made the Video component compatible with my existing code.